templates/angular/igx-ts/hierarchical-grid/hierarchical-grid-custom/index.ts

```diff
@@ -86,7 +86,7 @@ class IgxHierarchicalGridTemplate extends IgniteUIForAngularTemplate {
 			end: `</a>`
 		};
 
-		if (this.userExtraConfiguration["columnFeatures"]) {
+		if (this.userExtraConfiguration && this.userExtraConfiguration["columnFeatures"]) {
```

Comment:
I think it would be better if this was fixed in the beginning of the class, where the `userExtraConfiguration` is declared:
```typescript
class IgxHierarchicalGridTemplate extends IgniteUIForAngularTemplate {
	private userExtraConfiguration: {} = {};
       ...
}
```
This way, it's going to be in line with all of the other grid templates.